src/renderer/components/CustomEdge/CustomEdge.tsx

```diff
@@ -92,67 +92,42 @@ export const CustomEdge = memo(
                 onMouseOver={() => hoverTimeout()}
             >
                 <path
-                    className="edge-chain-links"
+                    className={useMemo(
+                        () =>
+                            `edge-chain-links ${isHovered ? 'hovered' : ''} ${
+                                showRunning && animateChain ? 'running' : ''
+                            }`,
+                        [isHovered, showRunning, animateChain]
+                    )}
                     d={edgePath}
                     fill="none"
                     id={id}
-                    strokeDasharray="0 !important"
-                    style={{
-                        ...style,
-                        strokeWidth: isHovered ? '4px' : '2px',
-                        stroke: currentColor,
-                        transitionDuration: '0.15s',
-                        transitionProperty: 'stroke-width, stroke',
-                        transitionTimingFunction: 'ease-in-out',
-                        cursor: 'pointer',
-                        strokeDasharray: '0 !important',
-                    }}
+                    stroke={currentColor}
                 />
                 <path
-                    className="edge-chain"
+                    className={useMemo(
+                        () =>
+                            `edge-chain ${isHovered ? 'hovered' : ''} ${
+                                showRunning && animateChain ? 'running' : ''
+                            }`,
```

Comment:
It might be worth to extract the `${isHovered ? 'hovered' : ''} ${showRunning && animateChain ? 'running' : ''}` part into a variable.